Portfolio.html

```diff
@@ -4,11 +4,85 @@
+    <link rel="stylesheet" type="text/css" href="normalize.css">
+    <link rel="stylesheet" type="text/css" href="Media-Query.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Rob Reeves Portfolio</title>
 </head>
-<body>
-
-
 
+<div>
+    <nav class="main-nav">
+        <div class="nav-button-left">
+            <a class="nav-text" href="#C1">ABOUT ME</a>
+            <a class="nav-text" href="#C3">CONTACT</a>
+        </div>
+        <div class="nav-button-right">
+            <a class="nav-text" href="#C2">THREE PEDALS ONLY</a>
+        </div>
+    </nav>
+        <div class="main-header-wrapper">
+            <div class="main-header-centre">
+                <h1>ROB REEVES</h1>
+                    <div class="sub-heading-wrapper">
+                        <div class="sub-heading-rule-before"></div>
+                            <p class="sub-heading">THIS WEBSITE IS ALL ABOUT ME!</p>
+                        <div class="sub-heading-rule-after"></div>
+                    </div>
+                    <div class="sub-heading-wrapper2">
+                        <p class="sub-heading2">There are lots of interesting things about me on this website. Please take your time and enjoy your stay. If you have any queries about anything then please don't hesitate to drop me a line.</p>
+                    </div>
+            </div>
+        </div>
+    <div class="main-content" id="C1">
+        <div class="main-content-wrapper">
+            <div class="sub-heading-rule-before"></div>
+                <h2>ABOUT ME</h2>
+            <div class="sub-heading-rule-after"></div>
+            <p class="main-content-text">I am a junior web developer with a passion for design and agile development. I am currently learning HTML, CSS, Javascript, PHP, and MySQL. I am also familiar with Git and other software environments. I am a certified scrum master and will be applying for my PHP Zend certification after I am graduated from Mayden Academy.</p>
+            <div class="article">
+                <img width="" height="" src="a-drop-of-water-osa-water-leaf-158725.jpeg" class="image" alt="sea shore">
+                <p class="main-content-text">Lots of interesting stuff about me, all of which is indeed very interesting and it will be so interesting that you will not fall asleep rots of interesting stuff about me, all of which is indeed very interesting and it will be so interestingots of interesting stuff about me, all of which is indeed very interesting and it will be so interestingots of interesting stuff about me, all of which is indeed very interesting and it will be so interestingots of interesting stuff about me, all of which is indeed very interesting and it will be so interestingots of interesting stuff about me, all of which is indeed very interesting and it will be so interesting</p>
+            </div>
+        </div>
+    </div>
+</div>
+    <div class="blog-wrapper">
```

Comment:
Indentation to this whole section lines 49 - 64 is not correct.

Also consider using a section in place of a div

Reply:
done.